Win32Emu/Win32/Modules/Kernel32Module.cs

```diff
@@ -97,6 +97,30 @@ public bool TryInvokeUnsafe(string export, ICpu cpu, VirtualMemory memory, out u
 			case "GETVERSIONEXW":
 				returnValue = GetVersionExW(a.UInt32(0));
 				return true;
+			case "VERSETCONDITIONMASK":
+				{
+					// VerSetConditionMask takes a ULONGLONG (64-bit) as first parameter
+					// We need to read it as two 32-bit values from the stack
+					var conditionMaskLow = a.UInt32(0);
+					var conditionMaskHigh = a.UInt32(1);
+					var conditionMask = ((ulong)conditionMaskHigh << 32) | conditionMaskLow;
+					var result = VerSetConditionMask(conditionMask, a.UInt32(2), a.UInt32(3));
+					returnValue = (uint)result;
+					_cpu!.SetRegister("EDX", (uint)(result >> 32));
+					return true;
+				}
+			case "VERIFYVERSIONINFOW":
+				{
+					// VerifyVersionInfoW also takes a DWORDLONG (64-bit) parameter
+					var conditionMaskLow = a.UInt32(2);
+					var conditionMaskHigh = a.UInt32(3);
+					var conditionMask = ((ulong)conditionMaskHigh << 32) | conditionMaskLow;
+					returnValue = VerifyVersionInfoW(a.UInt32(0), a.UInt32(1), conditionMask);
+					return true;
+				}
```

Comment:
Please put this into a seperate function decorated with the appropriate DllModuleExport attribute